lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -146,7 +172,7 @@ sub engine_workit($) {
         setpgrp(0, 0);
         $ENV{TMPDIR} = $tmpdir;
         log_info("$$: WORKING " . $job->{id});
-        if (open(my $log, '>', "autoinst-log.txt")) {
+        if (open(my $log, '>>', "autoinst-log.txt")) {
             print $log "+++ worker notes +++\n";
```

Comment:
so we have +++ setup notes… and then later one +++ worker notes with partially duplicated content. Can this be improved?